README.md

```diff
@@ -18,13 +18,13 @@ Include the library by adding an `implementation` reference to `dependencies` bl
 For [Java](https://mvnrepository.com/artifact/io.ably/ably-java/latest):
 
 ```groovy
-implementation 'io.ably:ably-java:2.0'
+implementation 'io.ably:ably-java:1.2.18'
 ```
 
 For [Android](https://mvnrepository.com/artifact/io.ably/ably-android/latest):
 
 ```groovy
-implementation 'io.ably:ably-android:2.0'
+implementation 'io.ably:ably-android:1.2.18'
```

Comment:
The same here, it should be `2.0` 😉

Reply:
Here you go https://github.com/ably/ably-java/pull/849/commits/ac1222ee49e9ebb6941100f708369723df8afc1a